setup.py

```diff
@@ -8,7 +8,7 @@
     'pytest',
     'PyGObject-stubs',
     'mypy',
-    'pycodestyle',
+    'pycodestyle'
```

Comment:
i usually end all lines in a multi-line list with a comma so you can easily add items and reorder, but I guess that is a style thing.

Reply:
I normally don't but can see the advantages, changed